### ISSUE TITLE
Bug: Yarn v1 Classic install fails non-existing file /typings/css-tree

### ISSUE THREAD

**Issue by MikeMcC399** [eslint/css#58]
### Environment

ESLint version:  N/A
@eslint/css version: `0.4.0 `
Node version:  `v22.14.0` LTS
Yarn version:  `1.22.22`
Operating System: Ubuntu `24.04.2` LTS

This is a regression from @eslint/css version: `0.2.0`
Also reproducible on @eslint/css version: `0.3.0`

### Which language are you using?

stylesheet

### What did you do?

```shell
corepack enable yarn
cd $(mktemp -d)
corepack use yarn@1
yarn init -y
yarn add @eslint/css@0.4.0
```

### What did you expect to happen?

Yarn should install without error

### What actually happened?

Error message similar to:

```text
error Package "" refers to a non-existing file '"/tmp/tmp.43w0AuuO0J/typings/css-tree"'.
```

```text
$ corepack enable yarn
cd $(mktemp -d)
corepack use yarn@1
yarn init -y
yarn add @eslint/css@0.4.0
Installing yarn@1.22.22 in the project...

yarn install v1.22.22
warning package.json: No license field
info No lockfile found.
warning No license field
[1/4] Resolving packages...
[2/4] Fetching packages...
[3/4] Linking dependencies...
[4/4] Building fresh packages...

success Saved lockfile.
Done in 0.05s.
yarn init v1.22.22
warning The yes flag has been set. This will automatically answer yes to all questions, which may have security implications.
warning package.json: No license field
success Saved package.json
Done in 0.02s.
yarn add v1.22.22
[1/4] Resolving packages...
error Package "" refers to a non-existing file '"/tmp/tmp.43w0AuuO0J/typings/css-tree"'.
info Visit https://yarnpkg.com/en/docs/cli/add for documentation about this command.
```

### Link to Minimal Reproducible Example

See above 

### Participation

- [ ] I am willing to submit a pull request for this issue.

### Additional comments

- Related to  https://github.com/eslint/css/issues/56

**Comment by nzakas** [comment]
We don't need to track this separately from #56.

I also feel like people using Yarn v1 should upgrade at this point.

**Comment by MikeMcC399** [comment]
@nzakas 

Some users are locked to Yarn v1 and it currently has 5 Million downloads / week. Although it is frozen, it has not been deprecated and there has been no end-of-life declared on it.

If Yarn v1 cannot be used to install `@eslint/css`, then the README should be updated to be clear that it is not supported.

**Comment by MikeMcC399** [comment]
- PR #83 resolved this issue
- PR #84 adds back the pm check for Yarn v1 Classic
